src/sdl2_wrapper.cpp

```diff
@@ -459,6 +459,40 @@ CAMLprim value resdl_SDL_GetDesktopDisplayMode(value vDisplay) {
   CAMLreturn(ret);
 };
 
+CAMLprim value resdl_SDL_GetDisplayBounds(value vDisplay) {
+  CAMLparam1(vDisplay);
+  CAMLlocal1(rect);
+
+  int displayIndex = Int_val(vDisplay);
+  SDL_Rect sdlRect;
+
+  SDL_GetDisplayBounds(displayIndex, &sdlRect);
+
+  rect = caml_alloc(4, 0);
+  Store_field(rect, 0, sdlRect.x);
+  Store_field(rect, 1, sdlRect.y);
+  Store_field(rect, 2, sdlRect.w);
+  Store_field(rect, 3, sdlRect.h);
+  CAMLreturn(rect);
+};
+
+CAMLprim value resdl_SDL_GetDisplayUsableBounds(value vDisplay) {
+  CAMLparam1(vDisplay);
+  CAMLlocal1(rect);
+
+  int displayIndex = Int_val(vDisplay);
+  SDL_Rect sdlRect;
+
+  SDL_GetDisplayUsableBounds(displayIndex, &sdlRect);
+
+  rect = caml_alloc(4, 0);
+  Store_field(rect, 0, sdlRect.x);
+  Store_field(rect, 1, sdlRect.y);
+  Store_field(rect, 2, sdlRect.h);
```

Comment:
Noticed one issue - `sdlRect.h` is here twice

Reply:
This one should be `sdlRect.w`

Reply:
Hmm, I was sure I fixed that. Good catch though!

Reply:
Oh, I fixed one function but not the other :facepalm: